_events/sr2020/virtual-competition-knockouts.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Virtual Competition Knockouts and Finals
+date: 2020-07-25 15:00:00
+layout: event
+type: competition
+location: Online
+---
+
+The Student Robotics Competition marks the ultimate stage of SR2020, showcasing the months of preparation. Our teams will work hard to push their virtual robots to the limit, competing against each other, and evolving their strategies to gain the upper hand.
+
+Code submission will be due at 10am. Our live-steam will run from 3pm to 4:30pm
```

Comment:
Do we need to indicate the length at this point? I think we have a few more things to finalise before we know that for sure (especially for the knockouts).